Cargo.toml

```diff
@@ -65,6 +65,12 @@ kona-protocol = { git = "https://github.com/op-rs/kona", tag = "kona-client/v0.1
 kona-registry = { git = "https://github.com/op-rs/kona", tag = "kona-client/v0.1.0-beta.13", default-features = false }
 kona-genesis = { git = "https://github.com/op-rs/kona", tag = "kona-client/v0.1.0-beta.13", default-features = false }
 
+# hana
+hana-oracle = { git = "https://github.com/celestiaorg/hana", rev = "ae45d20"}
+hana-celestia = { git = "https://github.com/celestiaorg/hana", rev = "ae45d20"}
+hana-host = { git = "https://github.com/celestiaorg/hana", rev = "ae45d20"}
+hana-client = { git = "https://github.com/celestiaorg/hana", rev = "ae45d20"}
```

Comment:
nit: Can we use a beta tag from `hana`, rather than a rev? Ask Diego to make a release